core/jazz_ui/src/config/templates/go-yaml.ts

```diff
@@ -1,8 +1,8 @@
 export const goTemplate = {
     template:
 `functions: 
-    function1 - go
-        handler - function1/index.handler
-    function2 - go
-        handler - function2/index.handler`
+    function1: 
+      handler: main
+    function2: 
+      handler: main`
```

Comment:
Here for all templates we are having both functions pointing to the same handler.

We will have to have different handler for each function.
For Go: main1, main2
For Java: com.slf.services.Handler1,  com.slf.services.Handler2
For Nodejs and Python: index.handler1, index.handler2

Reply:
Handler1 & Handler2 doesn't look like good names :) Why are we moving away from different folders?

Reply:
You mean different folders for each function i.e. function1/handler & function2/handler?

Reply:
which one should I change it to?
handler2 or function2/handler

Reply:
function1/handler & function2/handler, I think. Satish, can you please weigh in?

Reply:
Yep, different folders for different functions is a good structure in general.

Reply:
Let is just use: function1/handler & function2/handler?

Reply:
Made this change